collision_detection/include/moveit/collision_detection/collision_common.h

```diff
@@ -139,6 +159,7 @@ namespace collision_detection
     {
     }
     typedef std::map<std::pair<std::string, std::string>, std::vector<Contact> > ContactMap;
+    typedef std::map<std::string, DetailedDistance> DistanceDetailedMap;
```

Comment:
Should this Map mimic the one above?  Does it make sense to have:
`typedef std::map<std::pair<std::string, std::string>, DetailedDistance > DistanceDetailedMap;`
in which case, the detailed distance is given between all links/objects (in this case, the `nearest_object` element would be removed from `DetailedDistance`.